app/javascript/components/service-request-default/index.jsx

```diff
@@ -7,13 +7,21 @@ import createSchema from './service-request-default.schema';
 // NOTE: parameters to be used as filters
 let daysAgo;
 
+const getApprovalStateCheckboxesDefault = (miqRequestInitialOptions) => {
+  const defaultStates = [];
+  miqRequestInitialOptions.states.forEach((state) => {
+    defaultStates.push(state.value);
+  });
+  return defaultStates;
+};
+
 // NOTE: processing the user selected filter values
 const onSubmitData = (values, miqRequestInitialOptions) => {
   // Request Date (created_recently)
   if (values.selectedPeriod) { // user selected
     daysAgo = values.selectedPeriod;
   } else { // default
-    daysAgo = miqRequestInitialOptions.timePeriods[0].value;
+    daysAgo = miqRequestInitialOptions.timePeriods[1].value;
```

Comment:
Keep miqRequestInitialOptions.timePeriods[1].value in an if loop checks to make sure those values exists.